arm-templates/ExampleWithProxySQL/template.json

```diff
@@ -0,0 +1,447 @@
+            }
+        },
+		"MonitoringUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "User name used to monitor the nodes"
+            }
+        },
+		"MonitoringUserLoginPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password for the user used to monitor the nodes"
+            }
+        }
+    },
+    "variables": {
+        "numberOfReplicas":1,
+        "enableAzureResources": true,
+        "location":  "[resourceGroup().location]",
+        "_artifactsLocation": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/mysql-paas-replication/",
+        "_artifactsLocationSasToken": "",
+        "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'salinuxvm')]",
+        "imagePublisher": "Canonical",
+        "imageOffer": "UbuntuServer",
+        "nicName": "myVMNic",
+        "addressPrefix": "10.0.0.0/16",
+        "subnetName": "Subnet",
+        "subnetPrefix": "10.0.0.0/24",
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressName": "myPublicIP",
+        "publicIPAddressType": "Dynamic",
+        "vmName": "MyUbuntuVM",
+        "vmSize": "Standard_DS1_v2",
+        "scriptsUri": "[uri(deployment().properties.templateLink.uri, '.')]",
+        "virtualNetworkName": "MyVNET",
+        "linuxConfiguration": {
+            "disablePasswordAuthentication": true,
+            "ssh": {
+                "publicKeys": [
+                    {
+                        "path": "[concat('/home/', parameters('adminUsername'), '/.ssh/authorized_keys')]",
+                        "keyData": "[parameters('adminPasswordOrKey')]"
+                    }
+                ]
+            }
+        },
+        "subnetRef": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('virtualNetworkName'), variables('subnetName'))]",
+        "nestedtemplateMySQL": "[uri(variables('_artifactsLocation'), concat('nested/mysql.json', variables('_artifactsLocationSasToken')))]",
+        "nestedtemplateMySQLReplica": "[uri(variables('_artifactsLocation'), concat('nested/mysqlReplica.json', variables('_artifactsLocationSasToken')))]",
+        "sourceServerId": "[resourceId('Microsoft.DBforMySQL/servers', parameters('serverName'))]",
+        "skuName": "[concat(if(equals(parameters('skuTier'),'GeneralPurpose'),'GP',if(equals(parameters('skuTier'),'Basic'),'B',if(equals(parameters('skuTier'),'MemoryOptimized'),'MO',''))),'_',parameters('skuFamily'),'_',parameters('vCPU'))]",
+        "sku": {
+            "name": "[variables('skuName')]",
+            "tier": "[parameters('skuTier')]",
+            "capacity": "[parameters('vCPU')]",
+            "size": "[parameters('skuSizeMB')]",
+            "family": "[parameters('skuFamily')]"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2017-05-10",
+            "name": "MySQLServer",
```

Comment:
Will you set the connection security? Like adding firewall rules?

Reply:
I added allow all services